lib/content-services/src/lib/document-list/components/library-name-column/library-name-column.component.ts

```diff
@@ -62,36 +58,29 @@ export class LibraryNameColumnComponent implements OnInit, OnDestroy {
 
     private onDestroy$ = new Subject<boolean>();
 
-    constructor(
-        private element: ElementRef,
-        private nodesApiService: NodesApiService
-    ) {}
+    constructor(private element: ElementRef, private nodesApiService: NodesApiService) {}
 
     ngOnInit() {
         this.updateValue();
 
-        this.nodesApiService.nodeUpdated
-            .pipe(takeUntil(this.onDestroy$))
-            .subscribe(node => {
-                const row: ShareDataRow = this.context.row;
-                if (row) {
-                    const { entry } = row.node;
-
-                    if (entry === node) {
-                        row.node = { entry };
-                        this.updateValue();
-                    }
+        this.nodesApiService.nodeUpdated.pipe(takeUntil(this.onDestroy$)).subscribe((node) => {
+            const row: ShareDataRow = this.context.row;
+            if (row) {
+                const { entry } = row.node;
+
+                if (entry === node) {
```

Comment:
Should we check equality by `id`? Instead of an object in general?

Reply:
I would suggest investigating separately, as it's out of the eslint scope, and there are sites that have nodeId, and some other node types, so may need careful refactoring